frontend/src/pages/RoadTripPage/PackingListPage/PackingListPage.jsx

```diff
@@ -1,9 +1,6 @@
 import { React } from "react";
+import PackingList from "../../../components/PackingList/PackingList";
 
 export default function PackingListPage() {
-  return (
-    <div>
-      <p>Packing List</p>
-    </div>
-  );
+    return <PackingList />;
```

Comment:
So technically everything in the PackingList.jsx file could be in here instead. Personally I like how it separates the two ideas: "this is the page and then this is the element on the page" but it also means this file has only one line of code. Thoughts?

Reply:
I like this!  If in the future we wanted to show other components on the page, it makes sense to have the page separated from its components.

Reply:
Ooh interesting, I like this idea! It makes sense to me, might get tricky if we want to nest pages etc inside there, thinking about how I did the map page, but I think this is clean